### DISCUSSION TITLE
Error while creating vpc and subnets using terraform

### DISCUSSION THREAD

**Discussion by Maheshdvp** [learndevopsonline/learndevopsonline#1679] [Q&A] [answered]
Hi team, I wanna create VPC , subnets using terraform but showing error in following file could you  please help me in that
workstation:54.224.156.197
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/288340fa-2d06-4364-b47e-78698ef683fa)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/7d71795e-3cc0-4f46-8d7c-082b132fa01c)

**Comment by sreedharm07** [comment, marked as answer]
it is expecting subnets value but its not reciving that
check all vars files form where its getting the subnet value
just check flow that values from tfvars file to main.tf was there (check in vars files) you missed in vars file to declare subnet variable.
